sdk/communication/azure-communication-chat/azure/communication/chat/_chat_client.py

```diff
@@ -77,7 +77,8 @@ def __init__(
         self._credential = credential
 
         self._client = AzureCommunicationChatService(
-            self._endpoint,
+            credential=self._credential,
```

Comment:
Should this be passed in here?
The generated code has this typed as an `AzureKeyCredential`, but the chat client has it typed as `CommunicationTokenCredential` - so has this been tested with the auth policy that's being passed in on line 82?

Reply:
It turns out I needed to add `security: Anonymous` to SWAGGER.md, and reverted this change and all other AzureCommunicationChatService instantiations